extensions/amazon-aws/package.json

```
@@ -2,7 +2,7 @@
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "aws",
   "title": "Amazon AWS",
-  "description": "Integrate your AWS account in Raycast",
+  "description": "Integrate your AWS account in Raycast, by Falcon and Hodglim",
   "icon": "command-icon.png",
   "author": "Falcon",
```

```suggestion
  "author": "Falcon",
  "contributors": [
    "Hodglim"
  ],
```

We're working on showing contributors, let's add it already 😄

---

Ah perfect, this has now been updated as suggested 👍